packages/integration-boilerplate-node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/integration-boilerplate-node",
-  "version": "2.2.0",
+  "version": "2.2.1-0",
```

Comment:
I think there has been another boilerplate release since you started working on this PR, just double check to make sure the versioning makes sense and is going in-order